### PR TITLE
Magazine-react-tarea-7

### PR DESCRIPTION
**Pull request by ValentinaAtehortuaZ**

### CODE REVIEW

File: Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import TitleBoxes from "./TitleBoxes";
+
+const Boxes = () => {
+  return (
+    <section className="container-text">
+      <div className="box-left">
```
Comment (emolinaab) [inline Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx:7]: @ValentinaAtehortuaZ this should be a section

File: Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <div className="text">
```
Comment (emolinaab) [inline Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx:8]: @ValentinaAtehortuaZ try to apply className="text" directly to h6 element

File: Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          <h6>SATURN</h6>
+        </div>
+      </div>
+
+      <div className="container-boxes">
```
Comment (emolinaab) [inline Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx:13]: @ValentinaAtehortuaZ this should be a ul

File: Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <div className="box">
```
Comment (emolinaab) [inline Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Boxes.jsx:14]: @ValentinaAtehortuaZ all these boxes should be li elements

File: Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import qr from "../assets/img/barcode1.gif";
+import TitleFooter from "./TitleFooter";
+
+const Footer = () => {
+  return (
+    <div>
```
Comment (emolinaab) [inline Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Footer.jsx:7]: @ValentinaAtehortuaZ you should avoid the div element and use footer as root element

**Review by emolinaab — CHANGES_REQUESTED** [review]

**Comment by ValentinaAtehortuaZ** [comment]
Hello Esteban, I already made all the corrections.

**Review by emolinaab — APPROVED** [review]

**Pull request merged**